stock_financial_risk/models/stock.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2016 Carlos Dauden <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, exceptions, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    def _action_done(self, cancel_backorder=False):
+        if not self.env.context.get("bypass_risk"):
+            moves = self.filtered(
+                lambda x: (
+                    x.location_dest_id.usage == "customer"
+                    and x.partner_id.commercial_partner_id.risk_exception
+                )
+            )
+            if moves:
+                raise exceptions.UserError(
+                    _("Financial risk exceeded in partner:\n%s")
+                    % moves.mapped("partner_id.name")
+                )
+        return super(StockMove, self)._action_done()
```

Comment:
Nitpick: can skip passing arguments to `super` (same for `StockPicking` methods)
```suggestion
        return super()._action_done()
```